pylutron/__init__.py

```diff
@@ -841,18 +841,127 @@ def handle_update(self, args):
     return False
 
 
-class MotionSensor(object):
+class PowerSource(Enum):
+  """Enum values representing power source, reported by queries to
+  battery-powered devices."""
+  UNSET = -1
+  
+  # Values from ?HELP,?DEVICE,22
+  UNKNOWN = 0
+  BATTERY = 1
+  EXTERNAL = 2
+
+  
+class BatteryStatus(Enum):
+  """Enum values representing battery state, reported by queries to
+  battery-powered devices."""
+  UNSET = -1
+  
+  # Values from ?HELP,?DEVICE,22 don't match the documentation, using what's in the doc.
```

Comment:
do the below values make actually represent truth from the device? Would need to induce low battery :)

Reply:
The battery values have been very confusing. I'm really not sure what the truth is with these, and this is mostly about getting the mechanics right while the enum values are semi-guesses. Should I leave it out?